packages/itwin/one-click-lca-widget/package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "@itwin/one-click-lca-widget-react",
+  "version": "0.1.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Carbon Calculation.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "iTwin.js",
+    "Carbon Calculation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/itwin/viewer-components-react/tree/master/packages/itwin/one-click-lca-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/one-click-lca-widget.js",
+  "module": "lib/esm/one-click-lca-widget.js",
+  "typings": "lib/cjs/one-click-lca-widget",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "rebuild": "npm run clean && npm run build",
+    "test": "npm run build && npm run webpack:tests && npm run certa",
+    "webpack:tests": "webpack --config webpack.config.js 1>&2",
+    "certa": "certa -c certa.json -r chrome",
+    "cover": "nyc npm test"
+  },
+  "dependencies": {
+    "@itwin/insights-client": "^0.1.0",
+    "@itwin/itwinui-css": "^0.44.1",
```

Comment:
Can you make this a devDependency?

Reply:
or remove the dependency altogether? why are you using the css pkg directly?

Reply:
https://github.com/iTwin/viewer-components-react/pull/168#discussion_r798040961 
This was talked about before with the G&M widget.

Reply:
gah, we need to figure out why this is happening, its def a dep of itwinui-react so should be getting pulled in. lets remove this dep in all our pkgs, run `rush purge, then `rush update --full`, and then manually check the lock file for itwinui-css

Reply:
Gave it a try again with the G&M widget. Seems to be working now 🥴.